### PR TITLE
Cf feature/event data

### PR DESCRIPTION
**Pull request by chrisfenos**
- Deposit and Withdraw events displayed
- Progress component for event feedback

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -386,11 +398,16 @@ refrain entirely from distribution of the Library.
 If any portion of this section is held invalid or unenforceable under any
 particular circumstance, the balance of the section is intended to apply,
 and the section as a whole is intended to apply in other circumstances.
+>>>>>>> 1318cf2e6b607c3a2ded2b0bc038c656ba2a4c8d
```
Comment (5chdn) [inline LICENSE]: unresolved conflict committed

Comment (chrisfenos) [reply LICENSE]: @5chdn resolved

**Review by ChainSafeSystems — CHANGES_REQUESTED** [review]
please remove progress element before merge as we discussed.

File: src/screens/app/components/app/App.js [hunk 2]
```diff
@@ -23,41 +23,87 @@ class App extends Component {
...
+          goerliFromChain: _fromChain,
+        });
+      }
     });
-    executeDeposit(provider, amount, network, pubKey);
   }
 
+  getEventData = () => {
+    const { network, eventRecipient, eventValue, eventToChain, eventEvent, goerliRecipient, goerliValue, goerliFromChain } = this.state;
+    const eventObject = { eventRecipient, eventValue, eventToChain, eventEvent, goerliRecipient, goerliValue, goerliFromChain, network };
```
Comment (GregTheGreek) [inline src/screens/app/components/app/App.js]: this can be done in one line change `const` to `return`

**Review by GregTheGreek — CHANGES_REQUESTED** [review]

File: src/screens/app/components/app/App.js (hunk 2 above)
Comment (chrisfenos) [reply src/screens/app/components/app/App.js]: @GregTheGreek resolved

**Pull request merged**